src/libraries/System.Private.Xml/src/System/Xml/Xsl/Xslt/OutputScopeManager.cs

```diff
@@ -18,10 +18,10 @@ public struct ScopeReord
             public string nsUri;
         }
         private ScopeReord[] _records = new ScopeReord[32];
-        private int _lastRecord = 0;
-        private int _lastScopes = 0;  // This is cash of records[lastRecord].scopeCount field;
-                                      // most often we will have PushScope()/PopScope pare over the same record.
-                                      // It has sence to avoid adresing this field through array access.
+        private int _lastRecord;
+        private int _lastScopes;  // This is cash of records[lastRecord].scopeCount field;
+                                  // most often we will have PushScope()/PopScope pare over the same record.
+                                  // It has sence to avoid adresing this field through array access.
```

Comment:
```suggestion
        private int _lastScopes;  // Cache of records[lastRecord].scopeCount field;
                                  // most often we will have PushScope()/PopScope over the same record.
                                  // It makes sense to avoid accessing this field through the array.
```